Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v
     mv /app/build/bin/grpc_health_probe-linux-amd64 /app/build/bin/grpc_health_probe && \
     chmod +x /app/build/bin/grpc_health_probe
 
-FROM thorax/erigon:stable
```

Comment:
Hmmm why did we have `stable` in the master?

Reply:
Initial idea was to use to stable to smoothly overcome the Gibbs hard fork issues with BSC. This PR reverts that and the corresponding release/tag has been deleted and we are also not using the stable image with any of the nodes.

Reply:
I mean you could have a branch with `stable` in it but I see no point having it in `master`

Reply:
True. I was wrong to have it into `master` branch